### ISSUE TITLE
Test image is not normal

### ISSUE THREAD

**Issue by Cuthbert-Huang** [Wenxuan-1119/TransBTS#31]
I used your code to train 600 epochs on BraTS2020 dataset, but the background of the test image is red, how can this be?
![110](https://user-images.githubusercontent.com/86650676/150644904-8ae6a605-ff11-4f9c-b4c9-6bbae61a9fc9.png)

**Comment by NadiYa924** [comment]
you can  try  softmax_dice2,  ,you have to limit the label 0

**Comment by Cuthbert-Huang** [comment]
thank you very much

>

**Comment by WaterDhappy** [comment]
I have the same problem, have you solved it?

**Comment by H-CODE6** [comment]
Hello, do you have only two colors of the last output picture? I look forward to your reply.

**Comment by Lyangmli** [comment]
> 我用你的代码在BraTS2020数据集上训练了600个纪元，但是测试图像的背景是红色的，这怎么可能？ ![110](https://user-images.githubusercontent.com/86650676/150644904-8ae6a605-ff11-4f9c-b4c9-6bbae61a9fc9.png)
Have you solved the problem?

**Comment by Lyangmli** [comment]
Have you solved the problem?

**Comment by linda0227** [comment]
> you can try softmax_dice2, ,you have to limit the label 0
您好，我可以请教您一个问题吗，我在brats2020数据集的测试结果上传到在线验证平台上，提示我说没对齐，请问这个问题怎么解决？

**Comment by linda0227** [comment]
> I used your code to train 600 epochs on BraTS2020 dataset, but the background of the test image is red, how can this be? ![110](https://user-images.githubusercontent.com/86650676/150644904-8ae6a605-ff11-4f9c-b4c9-6bbae61a9fc9.png)

您好，我可以请教您一个问题吗，我在brats2020数据集的测试结果上传到在线验证平台上，提示我说没对齐，请问这个问题怎么解决？

**Comment by Zerkoar** [comment]
> 你可以尝试softmax_dice2，，你必须限制标签0

Can you provide a detailed explanation?
